### DISCUSSION TITLE
Equipment required

### DISCUSSION THREAD

**Discussion by jds3d** [mcguirepr89/BirdNET-Pi#291] [Q&A]
Is there a sticky somewhere listing all of the equipment required, maybe a recommended base setup?  I'm thinking about building a BirdNET-Pi but want to make sure I get the right stuff the first time.  First question is how much memory is required for the Raspberry Pi 4b?  They are sold out, so the resellers have jacked up the price and I don't want to buy one that's more expensive than necessary.

**Comment by jds3d** [comment]
I went to MicroCenter today and purchased a Raspberry Pi 400, which the sales rep said was the same as a 4b with 4gb ram; the only difference is that it's encased in a keyboard.  Trying to get everything installed on it now, but I'm running into some issues which I will post in a new thread.

**Reply by mcguirepr89** [reply]
The 400 model typically runs cooler, so that's good ~~ Let me know how your setup with the keyboard turns out -- I'm interested to see if this thing will live indoors or out? Or maybe you have some outdoor security cameras for which you can use the RTSP stream feature?:thinking:

**Reply by jds3d** [reply]
Now that I've got it running (just had to try the install again after Caddy worked out their issues), I plan to eventually build an outdoor enclosure.  For right now, I've got a mic pointing out an open window and everything is living happily indoors.  I do have some outdoor security cameras, but I have no idea how I would hook them up to the Rasperry Pi.  Also, I doubt we would see much since all of the interesting birds like to hide in the trees around here and I get a ton of house sparrows at my feeders.
